charts/beyla/templates/daemon-set.yaml

```diff
@@ -45,31 +42,6 @@ spec:
       {{- if .Values.priorityClassName }}
       priorityClassName: {{ .Values.priorityClassName }}
       {{- end }}
-      {{- if not (.Values.privileged) }}
```

Comment:
It would be prudent to get a second pair of eyes on these changes

Reply:
We probably could do it  in a separate PR IMO

Reply:
@marctc ok, I will revert this part